webview_panels/src/modules/documentationEditor/components/saveDocumentation/SaveDocumentation.tsx

```diff
@@ -26,7 +26,7 @@ const SaveDocumentation = (): JSX.Element | null => {
   const saveDocumentation = async () => {
     const result = (await executeRequestInSync("saveDocumentation", {
       ...currentDocsData,
-      updatedTests: isTestUpdatedForAnyColumn ? currentDocsTests : undefined,
+      updatedTests: currentDocsTests,
```

Comment:
i am assuming this is the only change which retains test by passing the current tests. rest all changes are for tests.

Reply:
yes, right @AdiGajbhiye